web/lib/domain-logic/flashcards.js

```diff
@@ -0,0 +1,140 @@
+import { useState, useEffect, useCallback } from 'react'
+import { supabase as sb } from './supabaseClient'
+import { useUser } from './auth'
+
+const TB = { FlashcardSets: 'flashcard_sets', Flashcards: 'flashcards' }
+const getConfig = (config = {}) => ({
+  count: 'exact',
+  returning: 'representation',
+  ...config,
+})
```

Comment:
`getConfig()` calls below don't pass any additional argument, so it's better to just define a fix `const config = {count: 'exact' ...}` rather than a function